Assets/Scripts/GameStats.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+
+public class GameStats
+{
+    private string nameSettingFile = "gameStats.json";  // Name of the stats file.
+
+    public List<LevelStats> levelStats;                 // List of level stats of each level.
```

Comment:
Are there any other data structures you considered for this, and if so, why did you discount them.

Reply:
To be honest, this is the first structure that came to mind. Using this structure daily in my development, I am more comfortable with it. But I think that other structures such as a dictionary or a hashtable could be more efficient in this case.